core/src/result.rs

```diff
@@ -31,6 +31,9 @@ pub enum Error {
     #[error("parsing failed: {0}")]
     Parser(String),
 
+    #[error("OverflowError: {0}")]
+    OverflowError(String),
```

Comment:
Now that we have a BinaryOperator <https://github.com/gluesql/gluesql/commit/53579312d1950e181ac81de231a2ef79668c0fbb> let's improve it with this.